proto/pbstatus/status.proto

```diff
@@ -15,11 +15,11 @@
 syntax = "proto3";
 
 package status;
-option go_package = "github.com/hashicorp/consul/proto/pbstatus";
 
 import "google/protobuf/any.proto";
 
 option cc_enable_arenas = true;
+option go_package = "github.com/hashicorp/consul/proto/pbstatus";
```

Comment:
Did you mean to remove this instead of moving it?

Reply:
Technically it doesn't actually matter as `buf` will ignore the directive. I have gone ahead and removed all the package directives as they really don't make sense for our uses.